pkgs/development/python-modules/quandl/default.nix

```diff
@@ -10,12 +10,12 @@
 
 buildPythonPackage rec {
   pname = "quandl";
-  version = "3.4.8";
+  version = "3.5.0";
```

Comment:
```suggestion
  version = "3.5.0";
  disabled = isPy27;
```
```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/mb5iwvg1pvg3azmb52x6485psgkagrhy-python2.7-quandl-3.5.0.drv' failed with exit code 1; last 10 log lines:
      This version of quandl no longer supports python versions less than 3.5.0. If you're
      reading this message your pip and/or setuptools are outdated. Please run the following to
      update them:

      pip install pip setuptools --upgrade

      Then try to reinstall quandl:

      pip install quandl
cannot build derivation '/nix/store/jwwvq1whm9921vv87jf77d481hp425hg-env.drv': 1 dependencies couldn't be built
[3 built (1 failed), 39 copied (109.9 MiB), 18.2 MiB DL]
error: build of '/nix/store/jwwvq1whm9921vv87jf77d481hp425hg-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/83372
1 package failed to build:
python27Packages.Quandl
```

Reply:
Using isPy3k b/c minimal diff, isPy27 not present.